graph/service.go

```diff
@@ -74,20 +74,26 @@ func (s *TagStore) CmdSet(job *engine.Job) error {
 	return nil
 }
 
-// CmdGet returns information about an image.
+type ImageDetail struct {
```

Comment:
I think it is just crap, use `s.ImageLookup` instead of `GetImageDetail` everywhere.